app/layout.tsx

```diff
@@ -32,11 +44,11 @@ export default async function RootLayout({
           {children}
           <footer className="flex items-center justify-between">
             <div className="flex gap-2 items-center">
-              {installer?.metadata?.copyright_markdown ? (
+              {metadata.copyright_markdown ? (
```

Comment:
I'm assuming we don't need the `?`s, because if we don't get the installer back the page won't render at all.